src/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:not(:nth-child(n+3)) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:not(:nth-child(n+4)) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:not(:nth-child(n+5)) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:not(:nth-child(n + 6)) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
You can combine several selectors (comma separated) if you have the same rule for each one:
```
.stars--1 .stars__star:first-child,
.stars--5 .stars__star:not(:nth-child(n+6)) {
  background-image: // your url
}
```

Reply:
I agree) Thx for help